common/vts.proto

```diff
@@ -11,109 +11,52 @@ import "attestation.proto";
 import "token.proto";
 
 
-enum TAType {
-    TA_UNSPECIFIED = 0;
-    TA_RAWPUBLICKEY = 1;
-    // Refer https://github.com/veraison/veraison/issues/76, to introduce certs here!
-    TA_INVALID = 255;
-}
-
-message EndorsementID {
-    AttestationFormat type = 1;
-    google.protobuf.Struct parts = 2;
-}
-
-message Evidence {
-    google.protobuf.Struct value = 1;
-}
-
-message GetEndorsementsRequest{
-    EndorsementID id = 1;
-    Evidence evidence = 2;
-}
-
 message Status {
     bool result = 1;
     string error_detail = 2;
 }
 
-message GetEndorsementsResponse {
-    Status status = 1;
-    string details = 2;
-    TrustVector trust_vector = 3;
-    google.protobuf.Struct endorsements = 4;
-}
-
-message SwComponentID {
-    AttestationFormat type = 1;
-    google.protobuf.Struct parts = 2;
-}
-
-message SwComponent {
-    SwComponentID id = 1;
-    google.protobuf.Struct attributes = 2;
+message Evidence {
+    google.protobuf.Struct value = 1;
 }
 
 message AddSwComponentsRequest {
-    repeated SwComponent info = 1;
+    repeated Endorsement sw_components = 1;
 }
 
 message AddSwComponentsResponse {
     Status status = 1;
 }
 
-message GetSwComponentRequest {
-    SwComponentID id = 1;
-}
-
-message GetSwComponentResponse {
-    Status status = 1;
-    google.protobuf.Struct attributes = 2;
-}
-
-message TrustAnchorID {
-    AttestationFormat type = 1;
-    google.protobuf.Struct parts = 2;
+enum EndorsementType {
+    UNSET = 0;
+    REFERENCE_VALUE = 1;
+    VERIFICATION_KEY = 2;
 }
 
-message TrustAnchorValue {
-    TAType type = 1;
-    google.protobuf.Struct value = 2;
-}
-
-message TrustAnchor {
-    TrustAnchorID id = 1;
-    TrustAnchorValue value = 2;
+message Endorsement {
```

Comment:
Also I am little nervous about the term Endorsement! Strictly speaking 
as per RATS Architecture, Endorsements are the entities which does not come in Evidence, but are elements used in Verification. 
Are we muddling up that distinction here ?

Reply:
There is no established term that we can use to group endorsed values, reference values, verification and identity keys.  I have picked Endorsement as it's the term used in the DICE "Endorsements" architecture.

Reply:
Also, we call the external interface [Endorsement Provisioning API](https://github.com/veraison/docs/blob/main/api/endorsement-provisioning/README.md).